ui/src/app/components/workflows/workflow-form/jobs/job/job-types/hyperdrive-job/hyperdrive-job.component.ts

```diff
@@ -68,10 +68,13 @@ export class HyperdriveJobComponent implements OnInit, OnDestroy {
         );
       }
     });
+    this.initHyperdriveType();
+  }
 
+  initHyperdriveType() {
```

Comment:
Actually, I think you could even make it better by returning the type and making this a function without side effects.

Reply:
Fixed